tests/test_strategy_batch.py

```diff
@@ -104,3 +104,33 @@ def test_protocol_best_subpop_strategy():
     assert state["mean"].shape == (5, 2)
     assert (state["members"][np.random.randint(5)] == state["members"][np.random.randint(5)]).all()
     assert (state["best_fitness"] == np.repeat(best_fitness, (5))).all()
+
+
+def test_protocol_global_best_strategy():
+    rng = jax.random.PRNGKey(0)
+    batch_strategy = BatchStrategy(
+        strategy_name="CMA_ES",
+        num_dims=2,
+        popsize=100,
+        num_subpops=5,
+        strategy_kwargs={"elite_ratio": 0.5},
+        communication='global_best',
```

Comment:
I am a bit confused here. You didn't add code for the option `global_best` to the protocol, right?

Reply:
I think I removed this already? I think this was accidentally pushed as something that was preparation for creating a `global_best` protocol as a follow-up